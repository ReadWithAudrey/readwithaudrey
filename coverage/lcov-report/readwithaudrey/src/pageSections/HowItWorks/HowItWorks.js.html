<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for readwithaudrey/src/pageSections/HowItWorks/HowItWorks.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">readwithaudrey/src/pageSections/HowItWorks</a> HowItWorks.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/3</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react'
import { TextBox } from '../../components'
import StyledHowItWorks from './HowItWorks.style'
import man from '../../images/manicon.svg'
import pair from '../../images/pairicon.png'
import book from '../../images/open-book-black.svg'
import PropTypes from 'prop-types'
&nbsp;
const HowItWorks = () =&gt; (
  &lt;StyledHowItWorks&gt;
    &lt;div id="step-1-register"&gt;
      &lt;h2 className="f2 pink tc"&gt;Register&lt;/h2&gt;
      &lt;img src={man} alt="Icon of a man" className="small-icon" /&gt;
      &lt;TextBox&gt;
        Tell us a little bit about yourself &amp; how you’d like to use Audrey
      &lt;/TextBox&gt;
    &lt;/div&gt;
    &lt;div id="step-2-connect"&gt;
      &lt;h2 className="f2 pink tc"&gt;Connect&lt;/h2&gt;
      &lt;img src={pair} alt="Icon of a pair of people" className="small-icon" /&gt;
      &lt;TextBox&gt;
        We will pair you with someone we think you will connect well with and
        help to facilitate your first reading session
      &lt;/TextBox&gt;
    &lt;/div&gt;
    &lt;div id="step-3-read"&gt;
      &lt;h2 className="f2 pink tc"&gt;Read&lt;/h2&gt;
      &lt;img src={book} alt="Icon of a man" className="small-icon" /&gt;
      &lt;TextBox&gt;
        Each reading takes roughly 10 minutes. Afterwards you’re welcome to
        discuss the experience
      &lt;/TextBox&gt;
    &lt;/div&gt;
  &lt;/StyledHowItWorks&gt;
)
&nbsp;
HowItWorks.propTypes = {
  siteTitle: PropTypes.string,
}
&nbsp;
export default HowItWorks
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 12 2018 12:51:26 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
