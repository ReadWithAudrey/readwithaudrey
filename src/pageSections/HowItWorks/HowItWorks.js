import React from 'react'
import { TextBox } from '../../components'
import StyledHowItWorks from './HowItWorks.style'
import man from '../../images/manicon.svg'
import pair from '../../images/pairicon.png'
import book from '../../images/open-book-black.svg'
import PropTypes from 'prop-types'

const HowItWorks = () => (
  <StyledHowItWorks>
    <div id="step-1-register">
      <h2 className="f2 pink tc">Register</h2>
      <img src={man} alt="Icon of a man" className="small-icon" />
      <TextBox>
        Tell us a little bit about yourself & how you’d like to use Audrey
      </TextBox>
    </div>
    <div id="step-2-connect">
      <h2 className="f2 pink tc">Connect</h2>
      <img src={pair} alt="Icon of a pair of people" className="small-icon" />
      <TextBox>
        We will pair you with someone we think you will connect well with and
        help to facilitate your first reading session
      </TextBox>
    </div>
    <div id="step-3-read">
      <h2 className="f2 pink tc">Read</h2>
      <img src={book} alt="Icon of a man" className="small-icon" />
      <TextBox>
        Each reading takes roughly 10 minutes. Afterwards you’re welcome to
        discuss the experience
      </TextBox>
    </div>
  </StyledHowItWorks>
)

HowItWorks.propTypes = {
  siteTitle: PropTypes.string,
}

export default HowItWorks
