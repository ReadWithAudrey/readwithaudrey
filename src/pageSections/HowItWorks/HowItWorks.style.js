import styled from 'styled-components'

const StyledHowItWorks = styled.div.attrs({
  className:
    'bg-light-green flex flex-column items-center justify-around ph4 pt4 pb5 montserrat w-100',
})``
export default StyledHowItWorks
