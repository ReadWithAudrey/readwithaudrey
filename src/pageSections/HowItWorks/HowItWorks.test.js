import React from 'react'
import renderer from 'react-test-renderer'
import HowItWorks from './HowItWorks'

describe('HowItWorks component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<HowItWorks />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
