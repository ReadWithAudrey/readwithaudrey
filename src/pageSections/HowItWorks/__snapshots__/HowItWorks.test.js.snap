// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowItWorks component snapshot test renders correctly 1`] = `
<div
  className="sc-kgoBCf bg-light-green flex flex-column items-center justify-around ph4 pt4 pb5 montserrat w-100 kVSTZj"
>
  <h2
    className="f2 pink tc"
  >
    Three steps to begin your reading adventure
  </h2>
  <div
    id="step-1-register"
  >
    <h2
      className="f3 pink tc"
    >
      Join
    </h2>
    <img
      alt="Icon of a man"
      className="small-icon"
      src="test-file-stub"
    />
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      Tell us a little about yourself and decide how you’d like to use Audrey e.g. as a reader or listener, or both.
    </p>
  </div>
  <div
    id="step-2-connect"
  >
    <h2
      className="f3 pink tc"
    >
      Connect
    </h2>
    <img
      alt="Icon of a pair of people"
      className="small-icon"
      src="test-file-stub"
    />
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      We’ll introduce you to your reading partner, share a hand-picked book, and help facilitate your first reading session.
    </p>
  </div>
  <div
    id="step-3-read"
  >
    <h2
      className="f3 pink tc"
    >
      Read
    </h2>
    <img
      alt="Icon of a man"
      className="small-icon"
      src="test-file-stub"
    />
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      Begin reading your chosen book together. Each reading takes 10-15 minutes. Enjoy interesting conversations with each other.
    </p>
  </div>
  <Link
    className="no-underline white w-100"
    to="/Form1/"
  >
    <button
      className="sc-gzVnrw bg-pink hover-bg-orange white pa2 br2 w-50 h3 f3 mt4 center giBLup"
    >
      Get started
    </button>
  </Link>
</div>
`;
